src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -14,8 +14,7 @@
      * числа 92, 18, 26 и 0.</p>
      */
     int[] getAllNumbers(String text) {
-        String s = text.replaceAll("[^0-9*?:.+\\-^\"_ ]", " ").replaceAll(" +\\s", " ");
-        s = s.replaceAll("^\\s*", "");
+        String s = text.replaceAll("[^0-9*?:.+-]+", " ").replaceAll("^[ ]", "");
```

Comment:
```suggestion
        String s = text.replaceAll("[^0-9*?:.+-]+", " ").trim();
```
try to user trim() instead of second replaceAll()